web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.test.tsx

```diff
@@ -111,7 +119,6 @@ test('startAgent re-throws errors that are thrown while spawning the process', a
   await act(async () => {
     [, error] = await result.current.startAgent();
   });
-  expect(error).toBeInstanceOf(AgentProcessError);
```

Comment:
After I added an actual implementation, this test was failing without the mock above. However, since it first checked only the class of the error, the failure message was largely unhelpful.

Instead, we now first check the details behind the error and only then check its class. This way the reported failure shows what the expected error message is vs what was actually returned.